src/servers/graphql-server.ts

```diff
@@ -101,13 +109,15 @@ const sessionContext = ({ token, ips, body, apiKey, apiSecret }) => {
         account = loggedInAccount
       }
 
-      addAttributesToCurrentSpan({ [ENDUSER_ALIAS]: domainUser?.username })
+      addAttributesToCurrentSpan({ [ENDUSER_ALIAS]: domainAccount?.username })
 
       return {
         logger,
         uid: userId,
         wallet,
+        // FIXME: we should not return this for the admin graphql endpoint
```

Comment:
What should we return? What is the fix? Is _this_ really the right place to add this information. Wouldn't it make more sense to add a comment to the admin/user resolver (ie. where it might potentially be (mis-)used) rather than here where it is being injected?

Reply:
every resolver get the `context` object, so this is not practical. this should only be used for the main graphql server. but the refactor can wait until api v1 is being deprecated/server removed.